### PR TITLE
core-intake ks revisions

### PR DESCRIPTION
**Pull request by kspurgin**

### CODE REVIEW

File: spec/core_helpers.rb [hunk 1]
```diff
@@ -22,13 +22,9 @@ def core_cache
 
   def populate_core(cache)
```
Comment (kspurgin) [inline spec/core_helpers.rb:23]: We only want one `populate_core` method in this file. All auth/vocab terms go in the same method, so terms can be reused across all core tests.

File: spec/core_helpers.rb [hunk 2]
```diff
@@ -22,13 +22,9 @@ def core_cache
...
     terms = [
+      ['personauthorities', 'person', '2020', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(20201599147149106)'2020'"],
       ['personauthorities', 'person', 'Kev', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(Kev1599058769862)'Kev'"],
-    ]
-    populate(cache, terms)
-  end
-
-  def populate_core(cache)
-    terms = [
+      ['personauthorities', 'person', 'Mark Smith', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(MarkSmith)'Mark Smith'"],
```
Comment (kspurgin) [inline spec/core_helpers.rb:27]: Some terms hadn't been added to the cache, so weren't getting transformed to refnames

File: spec/core_helpers.rb [hunk 3]
```diff
@@ -38,8 +34,8 @@ def populate_core(cache)
       ['personauthorities', 'person', 'Trevor', "urn:cspace:core.collectionspace.org:personauthorities:name(person):item:name(Trevor1599144536281)'Trevor'"],
       ['placeauthorities', 'place', 'Chillspot', "urn:cspace:core.collectionspace.org:placeauthorities:name(place):item:name(Chillspot1599145441945)'Chillspot'"],
       ['orgauthorities', 'organization', '2021', "urn:cspace:core.collectionspace.org:orgauthorities:name(organization):item:name(20211599147173971)'2021'"],
-      ['orgauthorities', 'organization', 'TIm Herod' "urn:cspace:core.collectionspace.org:orgauthorities:name(organization):item:name(TImHerod1599144655199)'TIm Herod'"],
-      ['orgauthorities', 'organization' 'Tesla', "urn:cspace:core.collectionspace.org:orgauthorities:name(organization):item:name(Tesla1599144565539)'Tesla'"],
+      ['orgauthorities', 'organization', 'TIm Herod', "urn:cspace:core.collectionspace.org:orgauthorities:name(organization):item:name(TImHerod1599144655199)'TIm Herod'"],
+      ['orgauthorities', 'organization', 'Tesla', "urn:cspace:core.collectionspace.org:orgauthorities:name(organization):item:name(Tesla1599144565539)'Tesla'"],
```
Comment (kspurgin) [inline spec/core_helpers.rb:38]: Some were missing commas or single quotes on the end

File: spec/fixtures/files/datahashes/core/intake1.json [hunk 4]
```diff
@@ -2,16 +2,16 @@
     "entryNumber": "IN2020.21",
     "entryDate": "2020-09-25",
     "entryReason": "consideration",
-    "entryMethod": "post;foundondoorstep",
```
Comment (kspurgin) [inline spec/fixtures/files/datahashes/core/intake1.json:5]: For vocabulary terms, input data should use the same form as between the single quotes at the end of the refname added to the cache

File: spec/fixtures/files/datahashes/core/intake1.json [hunk 5]
```diff
@@ -2,16 +2,16 @@
...
     "currentOwnerPerson": "2020",
     "currentOwnerOrganization": "2021",
     "depositorPerson": "Troy",
     "depositorOrganization": "Ninja",
     "depositorsRequirements": "All needed;Met",
     "approvalGroup": "board of trustees;executive committee;collection committee",
-    "approvalIndividual": "John Allen;Tim Joes",
-    "approvalStatus": "not approved; not required",
+    "approvalIndividual": "John Allen;Tim Joes;",
+    "approvalStatus": "not approved;;not required",
```
Comment (kspurgin) [inline spec/fixtures/files/datahashes/core/intake1.json:14]: There are three iterations of this group. Added delimiters to specify that. Definitely needed for approvalStatus to indicate blank value in 2nd group. Possibly not necessary for approvalIndividual, but good to be consistent/explicit

**Comment by kspurgin** [comment]
@kevin-kariuki -- after reviewing the changes I made to make tests pass, please merge this into your core-intake branch if you can. Then let me know it is ready.

**Review by kevin-kariuki — COMMENTED** [review]
Checked on the changes, tested locally and they are working as expected.

**Pull request merged**